rand_distr/Cargo.toml

```diff
@@ -30,4 +30,4 @@ rand_pcg = { version = "0.3.0", path = "../rand_pcg" }
 # For inline examples
 rand = { path = "..", version = "0.8.0", default-features = false, features = ["std_rng", "std"] }
 # Histogram implementation for testing uniformity
-average = "0.10.3"
+average = { version = "0.12", features = [ "std" ] }
```

Comment:
I think the "std" feature is not needed here.

Reply:
Without it I got errors claiming that `f64` doesn't have `sqrt()`.

Reply:
I see, that sounds like a bug in `average`.